src/styles/ant-vars.less

```diff
@@ -263,4 +263,24 @@ https://github.com/ant-design/ant-design/blob/master/components/style/themes/def
 
 // Tag
 @tag-default-bg: @allen-purple;
-@tag-default-color: @white-six;
\ No newline at end of file
+@tag-default-color: @white-six;
+
+// Popover
+// ---
+// Popover body background color
+@popover-bg: @dark-purple;
+// Popover text color
+@popover-color: #fff;
+// Popover maximum width
+@popover-min-width: 177px;
+@popover-min-height: 32px;
+// Popover arrow width
+@popover-arrow-width: 6px;
+// Popover arrow color
+@popover-arrow-color: @popover-bg;
+// Popover outer arrow width
+// Popover outer arrow color
+@popover-arrow-outer-color: @popover-bg;
+// Popover distance with trigger
+@popover-distance: 0px;
+@popover-padding-horizontal: @padding-md;
```

Comment:
I mainly just changed the colors here. You can see the original version here: https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less#L522